examples/src/test/scala/za/co/absa/atum/HdfsInfoIntegrationSuite.scala

```diff
@@ -0,0 +1,76 @@
+package za.co.absa.atum
+
+import org.apache.hadoop.fs.FileSystem
+import org.apache.log4j.LogManager
+import org.apache.spark.sql.{DataFrame, SaveMode}
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import za.co.absa.atum.model.{Checkpoint, Measurement}
+import za.co.absa.atum.persistence.ControlMeasuresParser
+import za.co.absa.atum.utils.SparkTestBase
+
+class HdfsInfoIntegrationSuite extends AnyFlatSpec with SparkTestBase with Matchers with BeforeAndAfterAll {
+
+  private val log = LogManager.getLogger(this.getClass)
+  val tempDir: String = LocalFsTestUtils.createLocalTemporaryDirectory("hdfsTestOutput")
+
+  override def afterAll = {
+    LocalFsTestUtils.safeDeleteTestDir(tempDir)
+  }
+
+  private val inputCsv = "data/input/wikidata.csv"
+  private def readSparkInputCsv(inputCsvPath: String): DataFrame = spark.read
+    .option("header", "true")
+    .option("inferSchema", "true")
+    .csv(inputCsvPath)
+
+  private def writeSparkData(df: DataFrame, outputPath: String): Unit =
+    df.write.mode(SaveMode.Overwrite)
+      .parquet(outputPath)
+
+  {
```

Comment:
I am not sure I get how this is styled. Shouldn't there be some keyword here?

Reply:
what what?
The {} block is used to limit the visibility of `val outputPath`, as a logical constrain. Or are you discussing the formatting of the block?

Reply:
Ah, interesting, haven't thought about it that way. I have never seen a standalone block like this in scala. So it was weird to me.